src/main/java/org/somerville/swag/display/Basket.java

```diff
@@ -12,6 +19,46 @@
 
     public Basket(JFrame oldFrame, Customer customer) {
 
+        /**
+         * Load and display pretend order at page construction
+         */
+
+        Product productA = new Product(0,"Product A","Description A", BigDecimal.valueOf(19.99), 5, "Path");
+        Product productB = new Product(1,"Product B","Description B", BigDecimal.valueOf(29.99), 10, "Path");
+        Product productC = new Product(2,"Product C","Description C", BigDecimal.valueOf(29.99), 10, "Path");
+        OrderLine lineA = new OrderLine(productA, 1);
+        OrderLine lineB = new OrderLine(productB, 5);
+        OrderLine lineC = new OrderLine(productC, 5);
+
+        Customer c = new Customer();
+        Order o = c.getCurrentOrder();
+        o.add(lineA);
+        o.add(lineB);
+        o.add(lineC);
+
+
+        List<OrderLine> lines = o.getOrderLines();
+
+        //---------------------TABLE LAYOUT-----------------------------------------------------------------------------
+
+        String[] columNames = {"Product" , "Quantity"};
+        DefaultTableModel model = new DefaultTableModel(columNames,0) {
+            @Override
+            public boolean isCellEditable(int row, int column) {
+                return false;
+            }
+        };
+
+        for (OrderLine orderLine : lines) {
```

Comment:
Glad you went with the Enhanced For Loop which provides increased readability and maintainability while iterating through a List.